build.sbt

```diff
@@ -69,7 +69,7 @@ lazy val sdkJava = project
       "Kalix Java SDK",
       "-noqualifier",
       "java.lang"),
-    Compile / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Server),
+    Compile / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Server, AkkaGrpc.Client), // FIXME we should only have client for TimerService
```

Comment:
> FIXME we should only have client for TimerService

Wouldn't that be difficult? why is that important?

Reply:
I think it will be difficult, indeed. 
This was more a remind to myself to see if something is possible.

Reply:
I removed it. As you said, that's not important.